build.sbt

```diff
@@ -66,7 +66,7 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
 
 lazy val coreJVM = core.jvm
   .configure(_.enablePlugins(JCStressPlugin))
-  .settings(replSettings ++ Seq(crossScalaVersions ++= Seq("0.13.0-RC1")))
```

Comment:
Oh, I didn't see that you already had 0.13.0-RC1 in the crossScalaVersions of coreJVM, it's probably better to keep it here then since the other crossScalaVersions will apply to projects that don't compile with Dotty yet.

Reply:
Hmm yes, I wasn't sure what you meant.
Adding it to stdSettings relies of .travis.yml logic to only launch the correct tasks. But that's kind of the case anyway.
But I agree with You, the intent in the previous approach was clearer. So I'll revert the last commit.